README.md

```diff
@@ -138,9 +139,6 @@ Hardfloat holds Chisel code that generates parameterized IEEE 754-2008 compliant
 floating-point units used for fused multiply-add operations, conversions
 between integer and floating-point numbers, and conversions between
 floating-point conversions with different precision.
-* **riscv-tools**
-([https://github.com/riscv/riscv-tools](https://github.com/riscv/riscv-tools)):
-We tag a version of the RISC-V software ecosystem that works with the RTL committed in this repository.
```

Comment:
Yes, as @Mixermachine mentioned, we do still tag a version, even if it's not a submodule.

Reply:
Would be great if this could be mentioned in the README.md
"Please checkout riscv-tools on your own. Use 
git clone *riscv-tools*
git checkout *can be found in file* "

Currently a new user will only get "There is no folder riscv-tools"

Reply:
Done. (The fact that a version is tagged is mentioned on line 34 above "The riscv-tools repository known to work with rocket-chip is noted in the file riscv-tools.hash")